lib/resty/kafka/request.lua

```diff
@@ -60,6 +60,15 @@ local function str_int64(int)
                 tonumber(band(int, 0xff)))
 end
 
+-- for easy to debug
+function mt.__tostring( self )
+    local str = ""
+    for _,v in ipairs(self._req) do
+        str = str .. tostring(v) .. " "
```

Comment:
First of all, I'm not very happy to add these code for debug, I think we'd better have these code on higher level.
But I'm fine if you optimized these code.